cli/tools/upgrade.rs

```diff
@@ -220,6 +229,7 @@ async fn fetch_and_store_latest_version<
         .current_time()
         .sub(chrono::Duration::hours(UPGRADE_CHECK_INTERVAL + 1)),
       last_checked: env.current_time(),
+      current_version,
```

Comment:
Nitpick—micro-optimization is to move this down here:

```rs
current_version: env.current_version().to_string()
```

Reply:
done